source/zh-TW/0.3.0/index.html.haml

```diff
@@ -123,14 +118,14 @@ version: 0.3.0
   ### 是否可以重寫更新日誌
   當然。哪怕已經上線了，也可以重新更新更新日誌。有許多開源項目更新日誌不夠新，所以作者就會幫忙更新。
 
-  另外，很有可能你忘記記錄一個重大功能更新。所以這時候應該去重寫更新日誌。
+  另外，很有可能你忘記紀錄一個重大功能更新。所以這時候應該去重寫更新日誌。
 
   ### 如何貢獻？
-  本文檔並不是**真理**。這只是原作者的個人建議，並且包括許多收集的例子。
+  本文件並不是**真理**。這只是原作者的個人建議，並且包括許多收集的例子。
   哪怕[本開源庫][gh]提供一個[更新日誌案例][CHANGELOG]，我刻意沒有提供一個
   過於苛刻的規則列表（不像[語義化版本格式][semver]）。
 
-  這是因為我希望通過社區達到統一觀點，我認為中間討論的過程與結果一樣重要。
+  這是因為我希望通過社群達到統一觀點，我認為中間討論的過程與結果一樣重要。
```

Comment:
英文原文是：
> This is because I want our community to reach a consensus. I believe the
discussion is as important as the end result.

建議翻譯為：這是因為我希望社群的人們有一致的信念，我的信念是是討論的過程跟產出的結果是一樣的重要。